README.md

```diff
@@ -2,3 +2,4 @@ Tpsx
 ====
 
 A plugin to get server TPS and MSPT by command.
+Use```/tpsx```to open or close.
```

Comment:
Please use `` `code` `` in inine code, and add spaces between code and common word.

Reply:
ok